src/ess/amor/load.py

```diff
@@ -142,21 +142,13 @@ def load_amor_ch_frequency(ch: RawChopper[RunType]) -> ChopperFrequency[RunType]
 def load_amor_sample_rotation(fp: Filename[RunType]) -> SampleRotation[RunType]:
     (mu,) = load_nx(fp, "NXentry/NXinstrument/master_parameters/mu")
     # For some reason this is length 1 and not scalar sometimes
```

Comment:
Comment outdated now?

Reply:
No it's still the case. It's just that we only read the first value.